sample/shared/src/commonMain/kotlin/com/mikepenz/markdown/sample/LicensesPage.kt

```diff
@@ -0,0 +1,45 @@
+package com.mikepenz.markdown.sample
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import com.mikepenz.aboutlibraries.Libs
+import com.mikepenz.aboutlibraries.ui.compose.m3.LibrariesContainer
+import com.mikepenz.aboutlibraries.ui.compose.m3.LibraryDefaults
+import com.mikepenz.markdown.sample.resources.Res
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+internal fun LicensesPage(
+    padding: PaddingValues,
+    modifier: Modifier = Modifier
+) {
+    var libs by remember { mutableStateOf<Libs?>(null) }
+    LaunchedEffect("aboutlibraries.json") {
+        libs = Libs.Builder()
+            .withJson(Res.readBytes("files/aboutlibraries.json").decodeToString())
+            .build()
+    }
+    LibrariesContainer(
+        libraries = libs,
+        modifier = modifier.fillMaxSize(),
+        // sample uses material 2 - proxy theme
```

Comment:
we can update the sample to use Material 3 instead

Reply:
I'm not familiar with using material3 components, can I ask for your help